content/news/2021-02-07-week-10-progess.md

```diff
@@ -19,9 +19,9 @@ This week:
 - I will be adding new sections to the same chapter with the same step by step approach – this will be my main task this week. All the (new) sections that I will be adding are subject to discussion. It is a work in progress 🙂
 - Finally, I will be making another YouTube tutorial introducing Mixxx to the first time user… This tutorial will be guided by [chapter 4 of the manual](https://manual.mixxx.org/2.2/en/chapters/library.html) and will go something like;
 
-`Start the Mixxx software`
-`Import music to the Mixxx library - import your audio files`
-`Analyze your library tracks (what it means to analyze a track)`
-`Configure Sound output - especially if you're not using any DJ Hardware`
-`Play your first track with Mixxx`
+`Start the Mixxx software` <br>
+`Import music to the Mixxx library - import your audio files` <br>
+`Analyze your library tracks (what it means to analyze a track)` <br>
+`Configure Sound output - especially if you're not using any DJ Hardware` <br>
+`Play your first track with Mixxx` <br>
```

Comment:
```suggestion
  - `Start the Mixxx software` <br>
  - `Import music to the Mixxx library - import your audio files` 
  - `Analyze your library tracks (what it means to analyze a track)` 
  - `Configure Sound output - especially if you're not using any DJ Hardware` 
  - `Play your first track with Mixxx` 
```
You can make nested lists in markdown by prefixing every line with two extra spaces.
https://commonmark.org/help/tutorial/10-nestedLists.html

Reply:
Also, the `<br>` tags are unneeded and backticks are for code.

Reply:
Thank you! :)